index.html

```diff
@@ -31,13 +32,213 @@ <h2 id="job-title">I am a Software Developer</h2>
+                    </div>
+                </div>
+            </section>
+            <section class="about-me">
+                <div id="about-div">
+                    <h2 id="about-me-title">About Me</h2>
+                </div>
+                <div id="about-me-description-div">
+                    <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#"><button id="get-resume-button">Get My Resume</button></a>
+                </div>
+                <p id="line3"></p>
+                <div class="languages-div">
+                    <div class="languages">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 55 (1).png" alt=""><p id="framework-text">Language </p></div>
+                            <ul class="individual-languages-ul">
+                                <li class="individual-languages">JavaSript</li>
+                                <li class="individual-languages">Ruby</li>
+                                <li class="individual-languages">Html</li>
+                                <li class="individual-languages">Css</li>
+                            </ul>
+                    </div>
+                    <div class="frameworks-div">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 56 (1).png" alt=""><p id="framework-text">Frameworks </p></div>
+                            <ul class="individual-frameworks-ul">
+                                <li class="individual-frameworks">Bootstrap</li>
+                                <li class="individual-frameworks">Ruby on Rails</li>
+                                <li class="individual-frameworks">RSPec</li>
+                                <li  class="individual-frameworks">CapyBara</li>
+                                <li class="individual-frameworks">Selenium</li>
+                            </ul>
+                            
+                        </ul>
+                    </div>
+                    <div class="skills-div">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 57 (1).png" alt=""><p id="framework-text">Skills </p></div>
+                           <ul class="individual-skills-ul">
+                            <li class="individual-skills">Codekit</li>
+                            <li class="individual-skills">GitHub</li>
+                            <li class="individual-skills">Codepen</li>
+                            <li class="individual-skills">Gitlab</li>
+                            <li class="individual-skills">Terminal</li>
+                           </ul> 
+                    </div>
```

Comment:
|Figma design|Your design|
|----------------|-------------------|
| ![Screenshot from 2022-12-05 10-21-29](https://user-images.githubusercontent.com/65129228/205587382-72a0ff2f-4bc2-4a4e-8d3c-4df3d0edd5a6.png) | ![Screenshot from 2022-12-05 10-22-31](https://user-images.githubusercontent.com/65129228/205587507-1755b805-f330-4957-8992-ea0a3753887a.png)|  

- [ ] This content holds a group of lists. Kindly use ul and li to structure them. You can use this structure as a start. :cool: 

***Example***

```
<ul>
  <li>
    Languages
    <ul>
      <li>
        Javascript
      </li>
      <li>
        HTML
      </li>
      <li>
        CSS
      </li>
    </ul>
  </li>
  <li>
    Frameworks
  </li>
  <li>
    Skills
  </li>
</ul>
```